MemeifyMe/app/src/main/res/layout/activity_main.xml

```diff
@@ -9,9 +9,52 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     tools:context=".MainActivity">
 
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="40dp"
+        android:layout_marginTop="20dp"
+        android:textAppearance="?android:attr/textAppearanceLarge"
+        android:text="Memifier"
+        android:id="@+id/titleTextView"
```

Comment:
I definitely prefer the snake_case used elsewhere in this XML -- should we stay in line with that? Also, again, let's go general -> specific (ex: tv_title).